integrations/ga-cross-domain-tracking.md

```diff
@@ -0,0 +1,38 @@
+# Google analytics cross-domain tracking
+
+> **Notice of usage:** Google Tag Manager (GTM) and Google Analytics 4 (GA4) are third party services and we provide this integration as is.
+We support a set of custom events and Data Layer variables for use with GTM and GA4, however we have no control over what happens to them and how they are used. The basic setup examples we provide have been tested and verified to work with the Mews Booking Engine, however if you need a more complex setup then we cannot provide the support to do so and we recommend to ask a specialist to set it up and test it for you.
+
+## What is cross-domain tracking?
+
+In Google Analytics 4 (GA4), cross-domain tracking is a way of tracking user activity across multiple domains, e.g. across a customer website and a shopping cart. Without configuring cross-domain tracking, Google will create new cookies for each visit and think that it is two users and two sessions, instead of just the same user in the same session.
+
+
+## Why is it important?
+
+Google Analytics 4 (GA4) requires cross-domain tracking configuration to accurately track user interactions that span across different domains. Without this configuration, a visitor who completes a booking with a payment through a widget hosted on another domain would be tracked incorrectly. This visitor would appear as two separate users in Google Analytics, resulting in disjointed sessions. Additionally, the referral source attribution would be compromised, potentially misrepresenting the origin of the traffic. Essentially, cross-domain tracking in GA4 ensures that the user journey and source data are correctly linked and reported, even when transactions or interactions occur across multiple domains.
+
+"For the booking engine widget, the `_gl` Google Linker query parameter has been implemented to maintain session continuity. Below is an illustration of how it functions."
```

Comment:
Is this bit of text meant to be here? It makes no sense to me and looks like it was added by mistake. What is the `_gl` Google Linker query parameter? Does it matter?

Reply:
It is the query parameter added programmatically to the URL when the redirect from Widget to Standalone Booking Engine is done. 

Maybe it is irrelevant here. It is information about "how it works under the hood".

Reply:
I deleted the `_gl`.